web-common/src/features/dashboards/time-dimension-details/time-dimension-data-store.ts

```diff
@@ -490,7 +490,8 @@ export const tableInteractionStore = writable<HighlightedCell>({
 });
 
 export const chartInteractionColumn = writable<ChartInteractionColumns>({
-  hover: undefined,
+  yHover: undefined,
+  xHover: undefined,
```

Comment:
**Scalability Issue**: Introduction of separate xHover and yHover states might lead to unnecessary state updates and potential performance bottlenecks when handling large datasets or high-frequency updates. <br> **Fix**: Consider consolidating hover states or implementing a throttling mechanism to limit the frequency of state updates. <br> **Code Suggestion**: 
 ```
 -  yHover: undefined,
 -  xHover: undefined,
 +  hover: { x: undefined, y: undefined },
 ```